scripts/trlite

```diff
@@ -29,6 +29,10 @@ if [ ! -d "$ZJS_BASE" ]; then
    exit 1
 fi
 
+if [ ! -v TRAVIS_COMMIT_RANGE ]; then
+    TRAVIS_COMMIT_RANGE=$(git log $(git log origin -1 --format='format:%H')..HEAD --format='format:%H')
```

Comment:
I'm pretty sure you can replace that nested $(git log ...) call with `origin/master`.